tests/ui/instrument-xray/flags-always-never-1.stderr

```diff
@@ -0,0 +1,2 @@
+error: incorrect value `always,never` for unstable option `instrument-xray` - either a boolean (`yes`, `no`, `on`, `off`, etc), or a comma separated list of settings: `always`, `never`, `ignore-loops`, `instruction-threshold=N`, `skip-entry`, `skip-exit` was expected
```

Comment:
The wording of the error doesn't make it clear that the problem is `always` and `never` are mutually exclusive. It could lead people to conclude that `ignore-loops,skip-entry,skip-exit` wouldn't be accepted either.

Reply:
Good point. I've added a bit of clarification in parentheses :+1: Since the error message is already quite long, it should be okay to make it longer. At the same time, I don't _quite_ like how complex it is, but I don't think separate options will make things easier.